test/controllers/admin/admins_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class Admin::AdminsControllerTest < ActionController::TestCase
+  attr_reader :admin
+
+  setup do
+    @admin = users(:admin)
+
+    sign_in_as(@admin)
+  end
+
+  test "GET index renders" do
+    sign_in_as(users(:admin))
+
+    get :index
+
+    assert_response :ok
+  end
+
+  test "GET show renders" do
+    get :show, id: @admin.id
+
+    assert_response :ok
+  end
+
+  test "POST create makes an approved admin" do
+    password = 'blabla'
+
+    assert_difference 'Admin.count', +1 do
+      post(:create, admin: {
+        email: 'admin@test.com',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.